content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst

```diff
@@ -0,0 +1,318 @@
+===================
+Production analysis
+===================
+
+.. |MO| replace:: :abbr:`MO (manufacturing order)`
+.. |MOs| replace:: :abbr:`MOs (manufacturing orders)`
+
+The *production analysis* report provides statistics about products manufactured using Odoo's
+*Manufacturing* app. The report is useful when trying to understand production costs, manufacturing
+durations, and other important statistics about manufactured products.
+
+To open the production analysis report, navigate to :menuselection:`Manufacturing app --> Reporting
+--> Production Analysis`.
+
+Search... bar
+=============
+
+The :guilabel:`Search...` bar appears at the top of the :guilabel:`Production Analysis` dashboard.
+Along with its drop-down menu, the :guilabel:`Search...` bar used to refine the statistics shown on
+the production analysis report.
+
+By default, the :guilabel:`Search...` bar includes the :guilabel:`End Date: [current year]` filter,
+so that data is only displayed for |MOs| completed within the current calendar year. To remove this
+filter, and show older data as well, click the :icon:`oi-close` :guilabel:`(Remove)` button on the
+right side of the filter.
+
+To only show the data for a specific product, type the product's name in the :guilabel:`Search...`
+bar, and press enter. Multiple products can be entered at the same time, so that only the data for
+those products is shown.
```

Comment:
What do you mean by multiple products can be entered at the same time? 
![image](https://github.com/odoo/documentation/assets/23611711/b123ccda-ff96-499f-8aae-6adc8bdb5db7)

Do you mean that multiple products can appear in the search results? As it's written, I think I'm interpreting the information as: multiple searches for products can be entered in the search bar